src/main/resources/js/condec.dialog.js

```diff
@@ -275,7 +346,40 @@
 				inputSummaryField.disabled = true;
 				selectLocationField.disabled = true;
 			}
+			conDecAPI.getDecisionGroups(id,documentationLocation,inputExistingGroupsField,selectLevelField,function(selectLevelField, inputExistingGroupsField, groups){
+				if(!(groups === null) && groups.length > 0){
+					var groupZero = groups[0];
+					if("High_Level" == groupZero){
+						selectLevelField.innerHTML= "";
+						selectLevelField.insertAdjacentHTML("beforeend", "<option selected  value='High_Level'>High Level</option>" +
+								"<option value='Medium_Level'>Medium Level</option>" +
+								"<option value='Realization_Level'>Realization Level</option>");
+					}else if("Medium_Level" == groupZero){

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)